.github/workflows/ci.yml

```diff
@@ -0,0 +1,109 @@
+name: Build and push
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  base-images:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        pg_version:
+          - "9.5"
+          - "9.6"
+          - "10"
+          - "11"
+          - "12"
+          - "13"
+          - "14"
+          - "15"
+
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push image
+        uses: docker/build-push-action@v5
+        with:
+          push: true
+          platforms: linux/amd64,linux/arm64
```

Comment:
It appears the [Postgres base container](https://hub.docker.com/_/postgres) supports [all of the following architectures](): linux/amd64, linux/arm32v5, linux/arm32v6, linux/arm32v7, linux/arm64v8, linux/i386, linux/mips64le, linux/ppc64le, linux/s390x... so why not go ***buck wild*** and support them all? :)

Reply:
I guess we might as well give it a shot and see how it goes.

@andyundso You up for that?  (maybe later, after you're happy with the other bits? :smile:)